assets/training/aoai/proxy_components/src/finetune_submit.py

```diff
@@ -43,24 +46,56 @@ def submit_finetune_job(self, training_file_id, validation_file_id, model, regis
         job_id = finetune_job.id
         status = finetune_job.status
 
-        # If the job isn't yet done, poll it every 30 seconds.
+        # If the job isn't yet done, poll it every 60 seconds.
         if status not in ["succeeded", "failed"]:
             logger.info(f'Job not in terminal status: {status}. Waiting.')
+            last_event = None
             while status not in ["succeeded", "failed"]:
-                time.sleep(30)
+                time.sleep(60)
                 finetune_job = self.aoai_client.fine_tuning.jobs.retrieve(job_id)
                 status = finetune_job.status
-                logger.info(f"current status of finetune job : {status}, polling after 30 sec")
-        else:
-            logger.debug(f'Fine-tune job {job_id} finished with status: {status}')
+                last_event = self._log_events(job_id, last_event)
```

Comment:
can we pass finetune_job object instead of job_id which would re fetch finetune_job over the network?

Reply:
we are calling 2 different apis in this case. for _log_metrics we can pass finetune_job object to avoid 1 extra call. done that